modules/script/script-impl/global.d.ts

```diff
@@ -0,0 +1,18 @@
+/**
```

Comment:
should probably be part of script-api module

Reply:
Commented above.